app/serializers/event_serializer.rb

```diff
@@ -5,7 +5,7 @@ class EventSerializer < ActiveModel::Serializer
 
   def date
     t = object.start_time
-    t.blank? ? '' : %{ #{t.time.strftime('%Y-%m-%dT%H:%M:%S')}#{t.formatted_offset(false)} }
+    t.blank? ? '' : %{ #{l t, format: :short}#{t.formatted_offset(false)} }
```

Comment:
undefined method `l' for #<EventSerializer:0x007fe87e8f5728>
Anyone else gets that?